boxwise_flask/models.py

```diff
@@ -24,5 +25,54 @@ def __unicode__(self):
 
     @staticmethod
     def get_camps():
-        # SELECT distinct id, organisation_id FROM dropapp_dev.camps;
         return Camps.select().order_by(Camps.name)
+
+
+class Cms_Usergroups_Camps(db.Model):
+    camp_id = CharField()
+    cms_usergroups_id = CharField()
+
+    class Meta:
+        primary_key = CompositeKey('camp_id', 'cms_usergroups_id')
+
+    def __unicode__(self):
+        return self.name
+
+    @staticmethod
+    def get_camp(usergroup_id):
+        camp = (Cms_Usergroups_Camps
+            .select(Cms_Usergroups_Camps.camp_id)
+            .where(Cms_Usergroups_Camps.cms_usergroups_id == usergroup_id)
+            )
+        return camp
```

Comment:
Is `get_camp_ids()` for a method name a little more precise? Also you could directly return the result of the query.